package.json

```diff
@@ -24,6 +24,9 @@
     "@angular/platform-browser-dynamic": "~8.2.14",
     "@angular/router": "~8.2.14",
     "@knora/api": "file:.yalc/@knora/api",
+    "@types/lodash": "^4.14.149",
```

Comment:
is this needed because of `_.isEqual`? If so, would there be other ways to make a comparison without an additional dependency?

Reply:
Removed in 046010e. I am only comparing strings now.